packages/main/src/themes/DateTimePicker.css

```diff
@@ -0,0 +1,3 @@
+:host .ui5-datepicker-input {
+	min-width: 14.5rem;
```

Comment:
I tested and for long formats (format-pattern="long"), like "Aug 18, 2020, 10:32:30 AM" it might still not be enough, 
14.75 to 15rem would be more safe:


<img width="569" alt="Screenshot 2020-05-28 at 2 51 28 PM" src="https://user-images.githubusercontent.com/15702139/83138000-ce2ff280-a0f2-11ea-8aa0-2496913140c1.png">


<img width="556" alt="Screenshot 2020-05-28 at 2 51 50 PM" src="https://user-images.githubusercontent.com/15702139/83138005-d0924c80-a0f2-11ea-8eac-dafd4d6983b4.png">

Reply:
Indeed, I tested the same but with "Apr". After your comment I noticed that "Aug" and "Sep" are longer in terms of pixels than "Apr" due to the variable-width font, so I increased to 15rem.

Reply:
This makes date-time picker width stand out when used alongside other inputs 
for example: 
![image](https://user-images.githubusercontent.com/6120010/226158973-17030ca6-a859-4ebf-b28c-8303738c08b1.png)